scripts/index-demo-page.js

```diff
@@ -165,6 +185,36 @@
     onload();
   };
 
+  var regenerateRepresentations = function() {
+    while (representationsEl.firstChild) {
+      representationsEl.removeChild(representationsEl.firstChild);
+    }
+
+    var selectedIndex;
+
+    window.vhs.representations().forEach(function(rep, i) {
+      var option = document.createElement('option');
+
+      option.value = rep.id;
+      option.innerText = JSON.stringify({
+        id: rep.id,
+        videoCodec: rep.codecs.video,
+        audioCodec: rep.codecs.audio,
+        bandwidth: rep.bandwidth,
+        heigth: rep.heigth,
+        width: rep.width
+      });
+
+      if (window.mpc.media().id === rep.id) {
+        selectedIndex = i;
+      }
+
+      representationsEl.appendChild(option);
+    });
+
+    representationsEl.selectedIndex = selectedIndex;
```

Comment:
This does not trigger a `change` event on `representationsEl` as that only happens when not using the js api.